doc/news/changes/minor/20190810MaeMarkowski

```diff
@@ -0,0 +1,3 @@
+New: Added the function subdivided_hyper_L to GridGenerator.
```

Comment:
This way, `doxygen` will be able to point directly to the new function.
```suggestion
New: Added the function GridGenerator::subdivided_hyper_L().
```

Reply:
@maemar Would you also take care of this before merging?

Reply:
Ah yes, sorry--I thought this was referring to the name of the PR on github. This is in the log file right?

Reply:
Exactly. This suggestion referred to your changelog entry :)